cockroachdb-in-comparison.md

```diff
@@ -2,3 +2,20 @@
 title: CockroachDB in Comparison
 toc: false
 ---
+
+| | CockroachDB | MySQL | Postgre | Cassandra | HBase | MongoDB | Riak 
+--|--|--|--|--|--|--|--
+Automated Scaling & Repair| Yes | No | No |  |  | No | Yes 
+Synchronous Replication | Yes | No | No |  |  | No |  
+Automatic Failover | Yes | No | No |  |  |  | Yes 
+Read from Any Replica | No |  |  |  |  | Yes |  
```

Comment:
@spencerkimball: One question on Riak. In your estimation, you say "No" for strong consistency, but they're docs say "Yes": http://docs.basho.com/riak/latest/dev/advanced/strong-consistency/. Thoughts?